openedx/features/idea/tests/test_favorite_api.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+
+from lms.djangoapps.onboarding.tests.factories import UserFactory
+from openedx.features.idea.tests.factories import IdeaFactory
+
+
+class IdeaFavoriteApiTestCase(TestCase):
+
+    def setUp(self):
+        super(IdeaFavoriteApiTestCase, self).setUp()
+        self.user = UserFactory()
+
+    def test_toggle_favorite_idea(self):
+        idea = IdeaFactory(
+            image='my_image.jpg',
+            file='my_file.docx',
+            video_link='https://example.com'
```

Comment:
Lets remove image, file, video_link, they are irrelevant for this test. Test will pass without these values